Source/Modules/ModuleEditor.h

```diff
@@ -42,4 +43,9 @@ class ModuleEditor : public Module
 inline const WindowScene* ModuleEditor::GetScene() const
 {
 	return scene;
-}
\ No newline at end of file
+}
+
+inline const std::unique_ptr<WindowDebug>* ModuleEditor::GetDebugOptions() const
+{
+	return &debugOptions;
```

Comment:
Once you change the member, this should be
```suggestion
return debugOptions.get();
```
And update the method signature to return a raw pointer.